pkg/expr/service_test.go

```diff
@@ -27,9 +27,9 @@ func TestService(t *testing.T) {
 		Frames: []*data.Frame{dsDF},
 	}
 	s := Service{
-		cfg:               setting.NewCfg(),
-		dataService:       me,
-		encryptionService: ossencryption.ProvideService(),
+		cfg:            setting.NewCfg(),
+		dataService:    me,
+		secretsService: fakes.NewFakeSecretsService(),
```

Comment:
nit, there's also a way to use real implementation with a fake secrets store - [example](https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/notifier/channels/pagerduty_test.go#L139)

Reply:
Sure, thanks!! 😊